.github/workflows/pr.yml

```diff
@@ -35,9 +35,9 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Build docker image for tests
-      run: docker build --target test -t basegun-back:tests backend/
+      run: docker build --target test --build-arg VERSION=1.0 -t basegun-back:tests backend/

```

Comment:
why?

Reply:
otherwise the VERSION variable is not defined. Now it's done at build time instead of putting it in the environment at runtime. I changed this a while ago. I forgot to update this test which explains why the CI failed in the previous commits